src/ngraph/serializer.cpp

```diff
@@ -2592,9 +2592,20 @@ shared_ptr<Node> JSONDeserializer::deserialize_node(json node_js)
         }
         case OP_TYPEID::Subtract:
         {
-            node = make_shared<op::Subtract>(
-                args[0], args[1], read_auto_broadcast(node_js, "auto_broadcast"));
-            break;
+            if (op_version == 0)
+            {
+                node = make_shared<op::Subtract>(
+                    args[0], args[1], read_auto_broadcast(node_js, "auto_broadcast"));
+                break;
+            }
+            if (op_version == 1)
+            {
+                node = make_shared<op::v1::Subtract>(
+                    args[0],
+                    args[1],
+                    read_auto_broadcast(node_js, "auto_broadcast", op::AutoBroadcastType::NUMPY));
+                break;
+            }
```

Comment:
Can you differ this serialization by `OP_TYPEID` instead of `op_version`?

Reply:
Sounds good.